Assets/Scripts/Combat/Tool_GridGenerator.cs

```diff
@@ -1,4 +1,5 @@
-﻿using UnityEditor;
+﻿#if (UNITY_EDITOR)
```

Comment:
What does this do?

Reply:
It tells the compiler to ignore the code if the environment isn't the Unity Editor - for example, when the project is built into an executable.